modules/auxiliary/admin/networking/cisco_dcnm_download.rb

```diff
@@ -7,62 +7,68 @@ class MetasploitModule < Msf::Auxiliary
 
   include Msf::Auxiliary::Report
   include Msf::Exploit::Remote::HttpClient
+  include Msf::Exploit::Deprecated
```

Comment:
Why is this now depreciated when it was released in 2019? I get some of these modules being redone as they are pretty old, but some of these like this one and the one from 2016 don't seem to make quite as much sense.

Reply:
This has now been resolved as this was a misunderstanding on my part. See discussion below.